napalm/ios.py

```diff
@@ -520,6 +520,109 @@ def get_interfaces(self):
 
         return interface_list
 
+
+    def get_interfaces_ip(self):
+        '''
+        Get interface ip details
+        Returns a dict of dicts
+
+        Example Output:
+
+        {   u'FastEthernet8': {   'ipv4': {   u'10.66.43.169': {   'prefix-length': 22}}},
```

Comment:
Quick question: if there are two logical units underneath FastEthernet8 for example (say 1 and 2), the output will contain two keys named FastEthernet8.1 and FastEthernet8.2?